client/sites-dashboard-v2/dotcom-style.scss

```diff
@@ -45,9 +45,6 @@
 	}
 
 	height: calc(100vh - var(--masterbar-height));
-	@media ( min-width: 601px ) {
-		height: calc(100vh - var(--masterbar-height) - 48px);
```

Comment:
It leads to overflow when the viewport is between 601px to 781px 

![image](https://github.com/Automattic/wp-calypso/assets/13596067/bb1cb6e8-f92b-4f73-bc9d-2d6d289c742b)


![image](https://github.com/Automattic/wp-calypso/assets/13596067/44d86c53-90d2-4d0c-991b-f04a9a0f2da0)

Reply:
Good catch, there are some styles from the sidebar that are hard to override 🙁